scipy/stats/_multivariate.py

```diff
@@ -2642,3 +2644,143 @@ def entropy(self):
     method_frozen.__doc__ = doccer.docformat(
         method.__doc__, wishart_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__, wishart_docdict_params)
+
+class special_ortho_group_gen(multi_rv_generic):
+    r"""
+    A matrix-valued SO(N) random variable.
+
+    The `dim` keyword specifies the dimension N.
+
+    Methods
+    -------
+    ``rvs(dim=None, size=1, random_state=None)``
+        Draw random samples from SO(N).
+
+    Parameters
+    ----------
+    dim : scalar
+        Dimension of matrices
+
+    TODO: frozen object not implemented yet.
+
+    Notes
+    -----
+    Return a random rotation matrix, drawn from the Haar distribution
+    (the only uniform distribution on SO(n)).
+
+    Examples
+    --------
+    >>> from scipy.stats import special_ortho_group
+    >>> x = special_ortho_group.rvs(5)
+
+    This generates one random matrix from SO(5).
+
```

Comment:
How about adding, for instance, `np.dot(x, x.T)`. (for display purposes you may want to do e.g. SO(3))